.cypress/integration/VisualizationCharts/12_pie_chart.spec.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import {
+  TEST_QUERIES,
+  querySearch,
+  landOnEventVisualizations,
+  saveVisualizationAndVerify,
+  deleteVisualization,
+} from '../../utils/event_analytics/constants';
+
+const renderPieChart = () => {
+  landOnEventVisualizations();
+  querySearch(TEST_QUERIES[3].query, TEST_QUERIES[3].dateRangeDOM);
+  cy.get('[data-test-subj="comboBoxInput"]').click();
+  cy.get('[data-test-subj="comboBoxOptionsList "] button span').contains('Pie').click({ force: true });
+};
+
+const aggregationValues = [
```

Comment:
I think these options are shared by other chart types as well, do we want to move to constants file?

Reply:
Good catch, moved it.